src/editors/playgroundController.ts

```diff
@@ -502,7 +541,19 @@ export default class PlaygroundController {
     });
   }
 
-  public deactivate(): void {
+  refreshPlaygroundResultDocument(
+    documentId: EJSON.SerializableTypes,
+    namespace: string
+  ): Promise<boolean> {
+    return new Promise(async (resolve) => {
+      return this._documentController.openEditableDocument(
```

Comment:
Doesn't look like the promise resolves, maybe we can just return the `this._documentController.openEditableDocument` result?

Reply:
Refactored the part about opening and saving mdb docs. Liked a lot your suggestion about using the in-memory file system 👍